### DISCUSSION TITLE
The site doesn't work

### DISCUSSION THREAD

**Discussion by jan2801** [ria-com/nomeroff-net#241] [General]

**Comment by ApelSYN** [comment]
The Nomeroff Net site is working fine. All models are stored here
https://nomeroff.net.ua/models/

Антон, если вы работаете на территории РФ, то возможно доступ к украинским IP заблокирован вашим государством.
В таком случае используйте vpn.

**Reply by ad-mos** [reply]
`requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://models.vsp.net.ua/config_model/nomeroff-net-np-classification/model-2.json`
При попытке запуска с примера.

**Reply by ApelSYN** [reply]
> `requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://models.vsp.net.ua/config_model/nomeroff-net-np-classification/model-2.json` При попытке запуска с примера.

Today we carried out some optimization work on the server that affected the correct operation of the web server, now I have fixed the problem.
We apologize for the inconsistent service.
